fuel/modules/fuel/libraries/Fuel_base_library.php

```diff
@@ -50,7 +50,7 @@ class Fuel_base_library {
 	 * @param	array	config preferences
 	 * @return	void
 	 */	
-	public function __construct()
+	public function __construct($config = array())
```

Comment:
Several sub-classes of this method pass in a config array, so it's best to have it in the parent class, too. Plus it already has a PHPDoc for it so I'm guessing this was accidentally left out.